v2/metrics.go

```diff
@@ -0,0 +1,66 @@
+package raftboltdb
+
+import (
+	"context"
+	"time"
+
+	metrics "github.com/armon/go-metrics"
+	"go.etcd.io/bbolt"
+)
+
+const (
+	defaultMetricsInterval = 5 * time.Second
+)
+
+func (b *BoltStore) RunMetrics(ctx context.Context, interval time.Duration) {
```

Comment:
A godoc string on this to mention "this is a long running function expected to be run in a goroutine, will exit when context is cancelled" might be good here.

Reply:
Good call. Its there now.